LOCAL_SANDBOX_TEST_GUIDE.md

```
@@ -0,0 +1,281 @@
+VERCEL_TOKEN=your_token_here
+
+# Ensure these are set (should already be in .env)
+ANTHROPIC_API_KEY=sk-ant-...
+LLM_PROVIDER=claude-agent-sdk  # Optional, defaults to openrouter
+```
+
+## Step 3: Verify Configuration
+
+```bash
+# Check environment variables are loaded
+grep VERCEL_TOKEN .env
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)